packages/store/test/unit/data-provider/telemetry-setting.test.ts

```diff
@@ -15,7 +15,7 @@ describe('TelemetrySetting data provider', () => {
         readAll: jest.fn()
     };
     beforeEach(() => {
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```

Comment:
Delete the line instead?

Reply:
Done in 3d67970ea55934680e91cd61e1b191bd65053b33